library/src/lib/localizator-editor/localization-editor-main/localization-editor-main.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, ContentChild, Input, OnInit, TemplateRef } from '@angular/core';
+import { LocalizationEditorInputDirective, LocalizationEditorLabel } from '../localization-editor.directives';
+
+/**
+ *  Component that represents main field with add-on and is always visible
```

Comment:
Component that represents the field which is always visible and is rendered outside the popover.